esn.php

```diff
@@ -258,8 +256,16 @@ function exception_error_handler($errno, $errstr, $errfile, $errline ) {
     $server->addPlugin(new ESN\CalDAV\Schedule\IMipPlugin($AMQPPublisher));
 }
 
+// Calendar scheduling support (must be after AMQP initialization)
+$server->addPlugin(
+    new ESN\CalDAV\Schedule\Plugin($principalBackend, $AMQPPublisher, $scheduleAsync)
+);
+
 $server->addPlugin(new ESN\CalDAV\Schedule\ITipPlugin());
 
+// IMIP callback endpoint for async processing
+$server->addPlugin(new ESN\CalDAV\Schedule\IMipCallbackPlugin());
+
 $server->addPlugin(new ESN\CalDAV\ParticipationPlugin());
 
 $server->addPlugin(new ESN\CalDAV\MobileRequestPlugin());
```

Comment:
Need a help
I’ve been debugging the iMIP flow and noticed something odd:
In `\ESN\CalDAV\Schedule\IMipPlugin::schedule()`, the variable `$isNewEvent` is always `false`,
even though `calendarObjectChange()` is executed right before and explicitly sets `$this->isNewEvent = true.` 
I don't understand why.